lib/smart_proxy_ansible/actions.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
-require 'foreman_tasks_core/shareable_action'
+require 'smart_proxy_dynflow/action/shareable'
```

Comment:
Why do we need this here?

Reply:
no idea why it was here, requires are very weird all over the place... I'll try to make them bit more sensible